site/content/integrate/apps/quickstart/apps-debug/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Using /apps debug command"
+heading: "Using /apps debug command"
+description: "`/apps debug` command can be used to debug an App's bindings, KV store records, view the logs, etc."
+weight: 100
+aliases:
+  - /integrate/apps/apps-debug/
+---
+
+This reference guide documents /apps debug command that can be used to display
+and modify apps' bindings, KV store records, and other data.
+
+To use this command you need to have both **Developer Mode** and **Testing Commands** set to `true`.
```

Comment:
Let's be a bit more explicit about where, in-product, users can set these values.

Reply:
In #1181 I've made this more clear.